src/main/resources/db/migration/V1.0__create_items_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE items(
```

Comment:
Since we haven’t merged this in yet, you can just name the table directly here instead of altering the table in a separate script. Same for the rename of the column and addition of the foreign key.